projects/step-frontend/src/lib/modules/timeseries/performance-view/measurements/measurements-picker.component.ts

```diff
@@ -20,13 +20,13 @@ export class MeasurementsPickerComponent implements OnInit, OnDestroy {
 
   terminator$ = new Subject<void>();
 
-  constructor(private executionPageService: TimeSeriesContextsFactory) {}
+  constructor(private contextsFactory: TimeSeriesContextsFactory) {}
```

Comment:
```suggestion
private contextsFactory = inject(TimeSeriesContextsFactory);
```

Reply:
I will refactor these and all similar in SED-2343 - in order to avoid conflicts. Hope it's fine.